src/Explorer/Panes/AddCollectionPanel.tsx

```diff
@@ -88,10 +88,10 @@ export interface AddCollectionPanelState {
   collectionId: string;
   enableIndexing: boolean;
   isSharded: boolean;
-  partitionKey: string;
+  partitionKeys: string[];
```

Comment:
Are the other APIs going to support hierachical partition keys in the future as well? If not, then I'm not a big fan of changing this to an array for other APIs too. If SQL is the only API that will have multiple partition keys, then we should create a separate property for SQL instead of forcing the partition keys of other APIs to be an array as well.